packages/data-portal-commons/src/components/ExpandableText.tsx

```diff
@@ -40,6 +40,11 @@ export const ExpandableText: React.FunctionComponent<ExpandableTextProps> = (
         </span>
     );
 
+    useEffect(() => {
+            setTruncated(true);
+          }, [props.fullText]);
```

Comment:
not sure why but the indentation looks incorrect here, can you make sure that the `prettier` hook is running properly?